### ISSUE TITLE
is it possible to connect more ultrasonic sensor 

### ISSUE THREAD

**Issue by nagendra092918** [SohamKorgaonkar/object_avoidance_in_drones#1]
i want implement same thing on four sides of drone,is it possible , if yes, how to do this

**Comment by SohamKorgaonkar** [comment]
> i want implement same thing on four sides of drone,is it possible , if yes, how to do this

Yes it is possible, the o in the function stands for orientation of the sensor
Refer to the Working in the readme

**Comment by nagendra092918** [comment]
tq for ur reply
